packit/base_git.py

```diff
@@ -415,6 +419,18 @@ def set_specfile_content(
             self.specfile.version = version
         if comment is not None:
             self.specfile.add_changelog_entry(comment)
+        if (
+            previous_version != self.specfile.expanded_version
+            and previous_release == self.specfile.release
+        ):
+            # This may occur if the upstream forgets to reset release after
+            # bumping the version, or if the specfile is not maintained in
+            # upstream at all (e.g. post-upstream-clone that uses wget to
+            # get the specfile from Fedora).
+            # Either way, it seems better to fix it at least in downstream
+            # at the cost of upstream and downstream having different
+            # specfiles.
+            self.specfile.release = 1
```

Comment:
Yeah, it works but good point, I will make it a string (since the `release` property returns a string)

Reply:
I don't know how I missed this. It works because it is being [implicitly converted to string](https://github.com/packit/specfile/blob/4eb120c060e1548e7bf6ae5f4d5bcd7de61d582a/specfile/tags.py#L483), but it's better if it's a string :slightly_smiling_face: